share/OpenMS/SCHEMAS/IdXML_1_4.xsd

```diff
@@ -249,7 +249,7 @@
 				<xs:documentation>significance threshold as calculated by the search engine</xs:documentation>
 			</xs:annotation>
 		</xs:attribute>
-		<xs:attribute name="spectrum_reference" type="xs:unsignedInt">
+                <xs:attribute name="spectrum_reference" type="xs:string">

```

Comment:
albeit, we [read](https://github.com/OpenMS/OpenMS/blob/develop/src/openms/source/FORMAT/IdXMLFile.cpp#L566) and [wrote](https://github.com/OpenMS/OpenMS/blob/develop/src/openms/source/FORMAT/IdXMLFile.cpp#L283) whatever was in there as string.